### PR TITLE
{math}[intel/2019b] FHI-aims v200112_2

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+postinstallcmds = ["cp -ar %(builddir)s/%(namelower)s.%(version)s/{CHANGELOG.md,doc,external} %(installdir)s/",
+                   "cp -ar %(builddir)s/%(namelower)s.%(version)s/{regression_tests,species_defaults} %(installdir)s/",
+                   "cp -ar %(builddir)s/%(namelower)s.%(version)s/{testcases,utilities} %(installdir)s/"]
+
+sanity_check_paths = {
+    'files': ['bin/aims.x'],
+    'dirs': [],
+}
+
+moduleclass = 'math'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb:49]: nitpicking: DFT is usually classified as `chem`, no?

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb (hunk 1 above)
Comment (dithwick) [reply easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb:49]: Sorry, was away. Yes I think so, although I've got engineers using it so didn't want to insult anyone :-) I will change it to chem and resubmit.

**Comment by dithwick** [comment]
Wondering if the test failed because this is licensed software so it can't be automatically downloaded?

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+particularly efficient for molecular systems and nanostructures, while
+maintaining high numerical accuracy for all production tasks. Production
+calculations handle up to several thousand atoms and can efficiently use (ten)
+thousands of cores. 
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'opt': True, 'precise': True}
+
+# The source code must be downloaded manually from the FHI-aims club.
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb:26]: @dithwick maybe worth mentioning that you need to sign and return the license form, for the download.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb (hunk 2 above)
Comment (dithwick) [reply easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb:26]: I'll expand the text, there are voluntary fees for the academic license so I don't want to phrase it in such a way that people skip this possible option.

Comment (dithwick) [reply easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb]: @verdurin I've expanded the text, is that the sort of thing you were looking for?

**Comment by dithwick** [comment]
Test report by @dithwick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
moorcroft.scrtp.warwick.ac.uk - Linux centos linux 7.8.2003, x86_64, Intel(R) Core(TM) i5-8600 CPU @ 3.10GHz, Python 2.7.5
See https://gist.github.com/9def6a55ac6cf4c0622323803ea0f924 for a full test report.

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @dithwick!

**Pull request merged**